docs/api-guide/serializers.md

```diff
@@ -774,6 +774,11 @@ Here's an example of how you might choose to implement multiple updates:
 
     class BookSerializer(serializers.Serializer):
         ...
+        id = serializers.IntegerField(
+            read_only=False,
+            required=False
+        )
```

Comment:
Not too sure about it being required. When I was trying to get it to work, creation would fail, as this field was required. hence, I made it not required.

Agreed that there is no need for explicit read_only=False.

How about:

```python
id = serializers.IntegerField(
            required=False
        )
```